homeassistant/components/mill/climate.py

```diff
@@ -222,7 +230,12 @@ def _update_attr(self) -> None:
         self._attr_target_temperature = data["set_temperature"]
         self._attr_current_temperature = data["ambient_temperature"]
 
-        if data["current_power"] > 0:
-            self._attr_hvac_action = HVACAction.HEATING
+        if data["operation_mode"] == "OFF":
```

Comment:
Should probably use `OperationMode.OFF` from upstream library here.

Reply:
Done